.eslintrc.cjs

```diff
@@ -15,16 +15,18 @@
  *
  * SPDX-License-Identifier: Apache-2.0
  */
-
```

Comment:
@BigBlueHat I recall the license tooling added these blank lines?  Do they need to be preserved?

Reply:
No, in fact, the licensing tool should not be inserting them.

Reply:
I don't think it was a choice, similar to the extra spaces in the copyright date range.  I was wondering if some part of that tooling would be grumpy if blank lines are gone.